pkg/cmd/server/start/config_test.go

```diff
@@ -76,19 +76,6 @@ func TestAssetPublicAddressDefaulting(t *testing.T) {
 	}
 }
 
-func TestAssetBindAddressDefaulting(t *testing.T) {
```

Comment:
nevermind... forgot this became a no-op when the console unified with the master